composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "c62c55b2f96b21c6942947c11925cc1f",
+    "content-hash": "4a063d7c5ba67c09c3ff597cc59e3620",
```

Comment:
It seems like the lock file has been updated by running `composer update`. That shouldn't happen since other dependencies are being updated like that. You should revert your lock file and then execute `composer update --lock` so that lock gets updated only by `.json` updates.